### PR TITLE
Add links to tokenizer API docs to refer relevant information.

### PR DESCRIPTION
**Pull request by BreakBB**
## Description
On [Gitter](https://gitter.im/explosion/spaCy) @johann-petrak asked about some Tokenizer information which are not linked to from the API docs.
This PR adds 2 links to the Tokenizer API docs so relevant information is a bit more accessible.

### Types of change
Doc change

## Checklist

- [x] I have submitted the spaCy Contributor Agreement.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/docs/api/tokenizer.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ tag: class
 source: spacy/tokenizer.pyx
 ---
 
-Segment text, and create `Doc` objects with the discovered segment boundaries.
+Segment text, and create `Doc` objects with the discovered segment boundaries. For a deeper understanding read about [how spaCy's tokenizer works](/usage/linguistic-features#how-tokenizer-works).
```
Comment (ines) [inline website/docs/api/tokenizer.md]:
```suggestion
Segment text, and create `Doc` objects with the discovered segment boundaries. For a deeper understanding, see the docs on [how spaCy's tokenizer works](/usage/linguistic-features#how-tokenizer-works).
```

**Comment by ines** [comment]
Thanks a lot! (I actually just create the suggestions so I can merge it myself, but for some reason, GitHub tells me I can't push to your branch?)

**Comment by BreakBB** [comment]
Hmmm... Can't tell why you could not push to my brnach 🤔 The checkbox about "Allow edits from maintainers" is still ticked.

**Comment by ines** [comment]
Strange! Maybe it was just GitHub acting up. Anyway, merging – thanks again 🙏

**Comment by BreakBB** [comment]
As always you're very welcome!

**Pull request merged**